lib/logs.py

```diff
@@ -86,13 +86,14 @@ def setup(dir='logs', minLevel=logging.INFO):
     stream_handler.setLevel(minLevel)
     stream_formatter = logging.Formatter(CONS_FORMAT)
     stream_handler.setFormatter(stream_formatter)
-    logger.addHandler(stream_handler)
+    # Redirect console to log file
+    logger.addHandler(file_handler)
```

Comment:
You've already set up a file_handler above in lines 77-82 for the purposes of DEBUG level logging.

I believe this removes logging from the console which was set to `minLevel` (which defaults to `INFO` per the `setup()` args, in line 54) , and redirects it to the same file_handler. I think this might actually cause duplication of INFO and higher level logs in that file since both file_handler and stream_handler logging handlers would be writing to it at the same time.